denguefever_tw/dengue_linebot/utils.py

```diff
@@ -62,17 +64,25 @@ def push_msg(line_bot_api, users, text, img):
         logger.info('Fail to push message!')
 
 
-def get_web_screenshot(zapper_id, web_url=ZAPPER_MAP_URL):
+def get_web_screenshot(zapper_id):
     logger.info('Getting zapper web screenshot and uploading......\n')
     display = Display(visible=0)
     display.start()
 
+    zapper_api = urljoin(BASE_ZAPPER_API_URL, 'lamps/{id}?key=hash'.format(id=zapper_id))
+    response = requests.get(zapper_api)
+    response_json = response.json()
+
     browser = webdriver.Chrome(executable_path=settings.CHROME_DRIVER_PATH)
     browser.set_window_size(1200, 900)
     browser.implicitly_wait(10)
-    browser.get(web_url)
+    browser.get('{url}?lng={lng}&lat={lat}'.format(
```

Comment:
You should use methods in `urllib` to handle URL instead of string formating.

Reply:
Fixed